src/backend/lang_c/c_simple_ast.rs

```diff
@@ -69,18 +76,20 @@ enum ValueEdge {
     GotoDst,
 }
 
-struct SimpleCAST {
+pub struct SimpleCAST {
     /// Name of function of this AST
     fname: String,
     /// Entry node of this function
     pub entry: NodeIndex,
+    /// Unknown node
+    pub unknown: NodeIndex,
```

Comment:
@kriw, I am a bit confused as to why this needs to be a default in a SimpleCAST structure?

Reply:
I think this is to have a single `unknown` index, which is later used in several places. @kriw please clarify.

Reply:
`unknown` is used as a default node which is used in the functions of `c_simple_ast_builder` so that  `var_map.get(node)` returns some value instead of `None`.